packages/data/walkthroughs/9.9.9.json

```diff
@@ -1555,8 +1555,230 @@
           "nextNavigateTo": "P11"
         }
       ]
-    }
-  },
+    }, 
+    "P18-9.9.9.3-remove-after-review": {
+      "walkthroughItemType": "multiChoice",
+      "questionText": "Is this dwelling unit located above another dwelling unit or common space?",
```

Comment:
We need to add `<defined-term>` around italicized terms like "dwelling unit".

`<defined-term>dwelling unit</defined-term>`